docs/customized-components/Button.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button as DSButton } from '../../src/index';
+import { Button as DSButton } from '../../packages/design-system/src/index'
```

Comment:
IMHO you shouldn't reach into packages likes this, monorepo is somewhat an implementation detail and you should access particular packages only through their respective names (in this example - `@livechat/design-system`).

You have two basic approaches that I know of to consume packages with build steps within your monorepo:
1. running watch to rebuild dist files which is really "meh" to me
2. aliasing, this unfortunately brings some extra config into the picture but at least each package can just reach for the newest version of other package and everything can somewhat be contained within a single process (in example webpack). We are using this approach and I've seen other people using it as well, I've also created a small helper package which can help with producing those aliases automatically - https://github.com/Andarist/lerna-alias

Reply:
done at 3638c3e1c30f75a3930047a8ae5c620a3bb40423